data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2194,10 +2278,38 @@
             x,y=36,20
         [/filter]
 
-        [message]
-            speaker=unit
-            message= _ "Ugh, a corpse. And it — (<i>gags</i>) — reeks!"
-        [/message]
+        [if]
+            [variable]
+                name=unit.id
+                equals="Abhai"
+            [/variable]
+            [then]
+                [message]
+                    speaker=Abhai
+                    message= _ "This corpse has been here for some time. I am glad I can no longer smell."
```

Comment:
I'd prefer to have the filter make this event not trigger for Abhai, but leave it active so that the next living unit can comment on it.

Given that he can no longer smell, it doesn't seem like something he'd think about, and "some time" would have to be a really long time in the case of Abhai.

Reply:
I agree. I'll just change it to filter out Abhai.